feat: Add HOWTO: Use the API to pull invoice data (#105)
citynetwork/docs: docs/howto/account-billing/pull-invoice-data.md

> # Pulling invoice data

Reviewer: ```suggestion
# Retrieving invoice data with the Cleura Cloud REST API
```

Author: Done!

Reviewer: Nope, you can't use macros here. :) They work fine in the page title, but they break navigation.

Author: Argh, I forgot about that :/

You may download some or all of your {{brand}} invoices using the

> {{rest_api}}.

Reviewer: Either include information of how to download the actual (PDF) *invoices* in this how-to, or else change "invoices" to "invoice data" or "invoice information".

Author: Makes much more sense, yes.

## Prerequisites

> Before downloading ---or pulling, if you will--- any of your invoices,

Reviewer: ```suggestion
Before retrieving any of your invoice data,
```

Author: Yes!

you need to
[create a token](/howto/getting-started/accessing-cc-rest-api)
for the current session.

## Saving invoice data

> To save all available invoices locally, type something like this:

Reviewer: ```suggestion
To save JSON data about all available invoices locally, construct a REST API request like this:
```

```bash

> curl -H "X-AUTH-LOGIN: your_username" \
>     -H "X-AUTH-TOKEN: your_token" \
>     -o invoices.json \
>     https://rest.cleura.cloud/account/v1/invoices

Reviewer: Maybe fix the indentation here:

```suggestion
curl -H "X-AUTH-LOGIN: your_username" \
     -H "X-AUTH-TOKEN: your_token" \
     -o invoices.json \
     https://rest.cleura.cloud/account/v1/invoices
```

Author: That was a tab-length effect, but yes, one more space is much better.

>      https://rest.cleura.cloud/account/v1/invoices

Reviewer: We should use the macro here.

Author: Ouch, yes we should :|

```

A command like the above will download the data of up to 20 invoices
and save them as JSON objects into the file `invoices.json`. You may
change the number of invoices with the `limit` parameter:

```bash
curl -H "X-AUTH-LOGIN: your_username" \
     -H "X-AUTH-TOKEN: your_token" \
     -o invoices.json \

>      https://rest.cleura.cloud/account/v1/invoices?limit=2

Reviewer: Here, too.

Author: Indeed! (Heavy British accent.)
